blackfire/blackfire.php

```diff
@@ -104,4 +104,22 @@ public function close() {}
      * @return string|null The sub-query or null if the current profile is not the first sample or profiling is disabled.
      */
     public function createSubProfileQuery() {}
+    
+    /**
+     * Sets a custom transaction name for Blackfire Monitoring.
+     *
+     * @param string $transactionName Name to register for the transaction (e.g. 'user_model:show')
+     *
+     * @return void
+     */
+    public function setTransactionName(string $transationName) {}
```

Comment:
Type in parameter name. That's why phpdoc is highlighted with "PhpDoc mismatch" inspection. Please correct the name.

Reply:
Thanks @isfedorov . Fixed now.